README.md

```diff
@@ -1,85 +1,95 @@
-
-#### The Toopher Two-Step
-Interacting with the Toopher web service involves two steps: pairing, and authenticating.
+### Python Version
+* 2.6
+* 2.7
 
-##### Pair
-Before you can enhance your website's actions with Toopher, your customers will need to pair their phone's Toopher app with your website.  To do this, they generate a unique, nonsensical "pairing phrase" from within the app on their phone.  You will need to prompt them for a pairing phrase as part of the Toopher enrollment process.  Once you have a pairing phrase, just send it to the Toopher web service and we'll return a pairing ID that you can use whenever you want to authenticate an action for that user.
+### Documentation
+Make sure you visit [https://dev.toopher.com](https://dev.toopher.com) to get acquainted with the Toopher API fundamentals.  The documentation there will tell you the details about the operations this API wrapper library provides.
 
-##### Authenticate
-You have complete control over what actions you want to authenticate using Toopher (for example: logging in, changing account information, making a purchase, etc.).  Just send us the user's pairing ID, a name for the terminal they're using, and a description of the action they're trying to perform and we'll make sure they actually want it to happen.
+## ToopherApi Workflow
 
-#### Librarified
-This library makes it super simple to do the Toopher two-step.  Check it out:
+### Step 1: Pair
+Before you can enhance your website's actions with Toopher, your customers will need to pair their mobile device's Toopher app with your website.  To do this, they generate a unique pairing phrase from within the app on their mobile device.  You will need to prompt them for a pairing phrase as part of the Toopher enrollment process.  Once you have a pairing phrase, just send it to the Toopher web service and we'll return a pairing ID that you can use whenever you want to authenticate an action for that user.
 
 ```python
 import toopher
 
-# Create an API object using your credentials
+# Create an API Object Using Your Credentials
 api = toopher.ToopherApi("<your consumer key>", "<your consumer secret>")
 
-# Step 1 - Pair with their phone's Toopher app
-pairing_status = api.pair("pairing phrase", "username@yourservice.com")
+# Step 1 - Pair with their mobile device's Toopher app
+pairing = api.pair("username@yourservice.com", "pairing phrase")
+```
+
+### Step 2: Authenticate
+You have complete control over what actions you want to authenticate using Toopher (for example: logging in, changing account information, making a purchase, etc.).  Just send us the user's pairing ID and a name for the terminal they're using and we'll make sure they actually want it to happen.
 
+```python
 # Step 2 - Authenticate a log in
-auth = api.authenticate(pairing_status.id, "my computer")
+authentication_request = api.authenticate(pairing.id, "terminal name")
```

Comment:
don't we want to encourage authentication using the username instead of pairing id?